piqe_ocp_lib/api/tasks/operator_ops.py

```diff
@@ -108,15 +108,49 @@ def add_operator_to_cluster(
         sub_resp = self.sub_obj.create_subscription(operator_name, channel_name, operator_namespace)
         return sub_resp is not None
 
-    def is_operator_installed(self, operator_name: str, operator_namespace: str) -> bool:
+    def is_operator_installed(self, operator_name: str) -> Optional[dict]:
```

Comment:
I don't think the type of this is `Optional`, assertion exceptions will be thrown and `None` is never explicitly returned. 

Also, using the `dict` builtin isn't supported until python 3.9+, you will need to import `Dict` from `typing` and use it like so:
```
from typing import Any
from typing import Dict
x: Dict[str, Any] = {"foo": {"bar": "cat"}, "test": ["food"]}
```

Note, `Any` is being used here as an example but if the type is able to be determined, it should be set. So `x` should really be `Dict[str, Union[Dict[str, str], List[str]]]` meaning the values inside of the dictionary can either be another dictionary of strings, or a list of strings. For more info https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html

You can also do this verification by executing `mypy`. This is how I did it for #57 

```
pip install mypy
mypy --ignore-missing-imports piqe_ocp_lib/api/resources/ocp_base.py
```

Reply:
Thanks, @jbpratt!